Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM scratch
```

Comment:
Any reason you picked this?

Reply:
I felt that we did not need any particular image since the binary is built outside of the container and its only purpose is to run the binary.

Thinking about it, having at least a busybox image could be useful for debug purposes wdyt?

Reply:
+1, I prefer to use something like busybox, scratch is painful when you have to debug something.

Reply:
Not sure what scratch comes with out of the box, but we should have at least some things like curl/wget that comes to mind for debugging yes. We can always change this later as well.

Reply:
[scratch](https://hub.docker.com/_/scratch) has nothing, it is an empty layer.